### ISSUE TITLE
very long starting time

### ISSUE THREAD

**Issue by samoht** [docker/datakit#287]
Seems that loading the full state can still take an awful lot of time:

```
2016-10-03 16:00 dkt-github [INFO] Loading full state
2016-10-03 16:00 dkt-github [INFO] Change detected in github-metadata-x
2016-10-03 16:21 dkt-github [INFO] API.add-webhook docker/datakit
```

We should be able to speed-up the 9p calls ...

**Comment by samoht** [comment]
Part of it should be fixed by #288 but more investigation should be done.

**Comment by samoht** [comment]
It's still not perfect but ocaml-git.1.9.1 improve the situation quite a bit, so closing.
